modules/Tvpapi/resources/tvpapiAnalytics.js

```diff
@@ -103,10 +103,12 @@
 		},
 
 		sendMediaHit: function() {
-			// Do not send media hit in the following conditions: concurrent limit, current time not updated (might be error)
-			// If isLive but not DVR then continue
-			if(this.concurrentFlag || (!(this.getPlayer().isLive()&&!this.getPlayer().isDVR())&&this.getPlayer().getPlayerElementTime() === 0)){
-				return;
+		    var isLive = this.getPlayer().isLive();
+		    var isDvr = this.getPlayer().isDVR();
+
+			// Do not send media hit in the following conditions: concurrent limit, current time not updated (might be error) when media is not (live and not dvr)
+			if(this.concurrentFlag || (!isLive && isDvr && this.getPlayer().getPlayerElementTime() === 0) ){
```

Comment:
@chanakabr not sure the condition here is correct - not isLive and is DVR - this condition cannot be met AFAICS

Reply:
@OrenMe  -  its ! (isLive && !isDVR) becoming  !isLive && isDVR. Right no?

Reply:
@OrenMe - Sorry typo. Fixing

Reply:
@OrenMe - Done.